rustic-cargo.el

```diff
@@ -143,9 +143,23 @@ stored in this variable.")
     map)
   "Local keymap for `rustic-cargo-test-mode' buffers.")
 
+(defvar rustic-test-failure
+  (let ((test "---- [^\n]+ ----\nthread [^\n]+ panicked at ")
```

Comment:
I'm not sure about including the initial `---- blah ---` bit, because tests that print to stdout will have their output below that but before the panic, e.g.:

```
---- entity::models::entity::insert_or_update_all_entity_types stdout ----
[lib/rs/spec-test/src/lib.rs:31:9] "TEST_ENVIRONMENT already set" = "TEST_ENVIRONMENT already set"
thread 'module::models::some_model::insert' panicked at lib/rs/lib-data/tests/lib-data-test/entity/models/entity.rs:89:53:
called `Result::unwrap()` on an `Err` value: EntityDbError(SqlxStoreError(Unexpected(Unexpected("failed to deserialize model: invalid type: null, expected string or map"))))
```

Reply:
Good point. But I'm not so sure that my approach is correct, anyway. I believe you're right in #573 that `rustic-compilation-panic` is not correct.